encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -49,21 +49,21 @@ function runTest(config,qualifier) {
             }).then(test.step_func(function() {
                 _events.push('updated');
                 if (event.messageType === 'license-release') {
-                    assert_array_equals(_events,
-                                [
-                                    'generaterequest',
-                                    'license-request',
-                                    'license-request-response',
-                                    'updated',
-                                    'keystatuseschange',
-                                    'playing',
-                                    'remove',
-                                    'keystatuseschange',
-                                    'license-release',
-                                    'license-release-response',
-                                    'closed-promise',
-                                    'updated'
-                                ],
+                    assert_equals(_events[0], 'generaterequest',"First event is generateRequest");
+                    var i = 1;
+                    for(; i < _events.length-8; i+=3 ) {
```

Comment:
I worry that the literal `8` could rot. One option would be to move the array at line 59 up as a variable and use its length here.

While less likely to change, `3` could also be problematic, and the same solution could be applied. (This would also address the formatting nit below.)